packages/editor/src/client/routes/peerList.tsx

```diff
@@ -54,6 +25,25 @@ import {PeerInfoEditPanel} from '../panel/peerProfileEditPanel'
 import {PeerEditPanel} from '../panel/peerEditPanel'
 
 import {useTranslation} from 'react-i18next'
+import {
+  Drawer,
+  FlexboxGrid,
+  List,
+  Avatar,
+  Icon,
+  IconButton,
+  Button,
+  Divider,
+  Modal,
+  Alert
+} from 'rsuite'
+import {DescriptionList, DescriptionListItem} from '../atoms/descriptionList'
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
Can you somehow avoid using @ts-ignore here?

Reply:
I would if i could :). Don't know how difficult it is. But i created an issue for this #76